provider/resources.go

```diff
@@ -275,6 +275,9 @@ func Provider() tfbridge.ProviderInfo {
 			"digitalocean_app": {
 				Tok: makeResource(digitalOceanMod, "App"),
 				Fields: map[string]*tfbridge.SchemaInfo{
+					"urn": {
+						Name: "appUrn",
+					},
```

Comment:
Nit: I think this looks cleaner and more readable.
```suggestion
					"urn": { Name: "appUrn" },
```